lib/ts/recipe/thirdparty/components/features/signInAndUpCallback/index.tsx

```diff
@@ -78,14 +81,20 @@ const SignInAndUpCallback: React.FC<PropType> = (props) => {
     const handleError = useCallback(
         (err) => {
             if (STGeneralError.isThisError(err)) {
-                return SuperTokens.getInstanceOrThrow().redirectToAuthWithoutRedirectToPath(undefined, props.history, {
-                    error: "custom",
-                    message: err.message,
+                return SuperTokens.getInstanceOrThrow().redirectToAuth({
+                    history: props.history,
+                    queryParams: {
+                        error: "custom",
+                        message: err.message,
+                    },
```

Comment:
need to explicitly set redirectBack to false -> internal use of this function should always explicitly set this value to prevent future bugs.